### DISCUSSION TITLE
Add 'run' command in PowerShell 7.3 Release update

### DISCUSSION THREAD

**Discussion by PSGitHubUser1** [PowerShell/PowerShell#16796] [Enhancements]
**It will be very easy to run a program by just typing 'run' and typing App name like - 'run type[app{Roblox}]' it will be nice..**

- [x] This is a good idea
- [ ] This is a bad idea

**Comment by kilasuit** [comment]
Did you know that you can do this already for any app that shows as a type of Application via `Get-Command -CommandType Application`  ??

**Comment by kasini3000** [comment]
It looks like you want to alias "&" with “run”.
